en/docs/guides/cleansing-data.md

```
@@ -195,9 +195,9 @@ To try out the above example, follow the steps below:
     select ifThenElse(deviceID is null, "UNKNOWN", deviceID) as deviceID, roomNo, temp
     insert into CleansedDataStream;
     ```
-   In this Siddhi application, the `Temp Stream` has an attribute named `deviceID`, but it is not selected to be included in the output events. The `roomNo`attribute is cast as an string value via `cast(roomNo, "string")`. This means although the value for this attribute is received as an integer, it is presented as a string value in the output.
+   In this Siddhi application, the `TempStream` stream has a `deviceID` attribute that may arrive with a null value. The `ifThenElse(deviceID is null, "UNKNOWN", deviceID)` expression substitutes `"UNKNOWN"` whenever `deviceID` is null and passes the original value through otherwise. The result is inserted into `CleansedDataStream` so downstream queries see a non-null `deviceID`.
    
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)